draft-ietf-tls-esni.md

```diff
@@ -787,6 +815,48 @@ where HKDF-Expand-Label and HKDF-Extract are as defined in {{RFC8446}}. The
 value of ServerHello.random[0:24] is generated as usual by invoking a secure
 random number generator (see {{RFC8446}}, Section 4.1.2).
 
+When confirming ECH acceptance, the backend server SHOULD reply to the
+"handshake_padding" extension solicited by the client, computing the padding
+length as described in {{server-padding}}.
+
+### Recommended Padding Scheme {#server-padding}
```

Comment:
I'd recommend pulling this out into a separate issue entirely. The client-side padding policy took a while to get consensus, so trying to spell the server-side story here might delay landing of the general mechanism. Can we do that?

Reply:
In that case, I think we should postpone this change until affter -08.

Reply:
Yeah, that seems reasonable.

Reply:
This PR already has an associated issue (https://github.com/tlswg/draft-ietf-tls-esni/issues/264). Why don't I just add a comment there?

Reply:
That's what I was thinking, yeah.